cli/stork-publisher-agent/README.md

```diff
@@ -0,0 +1,63 @@
+# Stork Publisher Agent
+
+The easiest way to become a Stork Publisher is to run the Stork Publisher Agent docker container on your infrastructure and send price updates to the Agent through a local websocket. The Stork Publisher Agent will sign your price updates with your private key and send them to the Stork Network.
+
+## Getting Started
+
+### Setup
+To run the agent on your infrastructure, you'll need to first create a `config.json` file containing all non-secret configuration and a `secrets.json` file containing all secret configuration, and then run the Stork Publisher Agent docker container.
+
+Here we open a websocket at `ws://localhost:5216/publish` which will both EVM sign every update.
+
+`config.json`:
+```json
+{
+  "SignatureTypes": ["evm"],
+  "IncomingWsPort": 5216,
+  "SignEveryUpdate": true
+}
+```
+
+You'll need to generate your own EVM and/or Stark keys, choose a unique 5 character oracle id and get a StorkAuth key from Stork.
```

Comment:
Note - It would be nice if we added a tool in the cli to generate keys eventually too